app/frontend/helpers/ezra_helper.js

```diff
@@ -18,8 +18,8 @@
 
 
 /**
- * This module contains utility functions that are used througt the app
- * @module ezra_helper
+ * This module contains utility functions that are used through the app
+ * @module ezraHelper
```

Comment:
I think using camelCase instead of under_line is more appropriate as in code we would use camelCase (i.e. ezdraHelper.waitUntilIdle())